score/volumes/README.md

```diff
@@ -1,5 +1,5 @@
 Define volumes and volume mounts for a Workload.
 
-Volume mounts are declared in the `volumes` section of the Score file. There should be a resource of type `volume` backing the volume.
+Volume mounts are declared in the `volumes` section of the Score file. There should be a resource of type `volume` defined in your resource definitions, [see examples here](https://developer.humanitec.com/examples/resource-definitions/volume-pvc/volumes/).
```

Comment:
This link will capture all Resource Definition examples containing a `type: volume`, regardless of its Driver.

```suggestion
Volume mounts are declared in the `volumes` section of the Score file. There should be a resource of type `volume` defined in your resource definitions, [see examples here](https://developer.humanitec.com/examples/resource-definitions?types=volume).
```